ingestion/functions/parsing/brazil_paraiba/paraiba.py

```diff
@@ -0,0 +1,189 @@
+import os
+import sys
+from datetime import date, datetime
+import csv
+
+# Layer code, like parsing_lib, is added to the path by AWS.
+# To test locally (e.g. via pytest), we have to modify sys.path.
+# pylint: disable=import-error
+try:
+    import parsing_lib
+except ImportError:
+    sys.path.append(
+        os.path.join(
+            os.path.dirname(os.path.dirname(os.path.abspath(__file__))),
+            'common/python'))
+    import parsing_lib
+
+_DATE_DEATH_INDEX = "Data do Óbito"
```

Comment:
nit but those are not "indexes" anymore, they are column names so please rename

Reply:
will change